string_manipulation/helper.py

```diff
@@ -0,0 +1,55 @@
+
+
+def repeating_to_single_string_converter(multi_occuring_string):
+    string_after_single_occurence = ""
+
+    for index_for_multi_occurence in multi_occuring_string:
+        if index_for_multi_occurence.lower() not in string_after_single_occurence.lower():
+            string_after_single_occurence = string_after_single_occurence + index_for_multi_occurence
+
+    return string_after_single_occurence
+
+
+def is_repeating(repeating_character_string, key_character):
+    counter = 0
+    is_true = True
+    for index_for_repeating_character in repeating_character_string:
+        counter = counter + 1 if index_for_repeating_character.lower() == key_character.lower() else counter
+
+    is_true = True if counter > 1 else False
+    return is_true
+
+
+def remove_given_character(input_string, key_character):
+    index_for_character_removal = input_string.find(key_character)
+
+    if index_for_character_removal == 0:
+        input_string = input_string[1:]
+    elif index_for_character_removal == len(input_string) - 1:
+        input_string = input_string[0: len(input_string) - 1]
+    else:
+        input_string = input_string[0:index_for_character_removal] + input_string[index_for_character_removal + 1: len(input_string)]
+
+    return input_string
+
+
+def remove_first_occurence_of_character(removing_string):
+    """How 'remove_first_occurence_of_character(arg)' work.
+
+       this function first uses 'repeat()' to find repeating characters and then uses
+       'remove_given_character(,)' to remove it's first occurrence
```

Comment:
Each function's description should be in its own docstring.
This docstring is for `remove_first_occurence_of_character` and should only contain description for `remove_first_occurence_of_character`

Reply:
Sir this docstring is already inside the method it's being used in, remove_first_occurence_of_character, please re verify